controllers/chat/deleteMessage.js

```diff
@@ -18,27 +18,27 @@ const deleteMessage = async (req, res) => {
     await client.partialUpdateMessage(messageID, {
       set: {
         message_type: MESSAGE_TYPE.DELETED,
-        text: 'This message has been deleted'
+        text: 'This message has been deleted',
+        attachments: []
       }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code is setting the `text` and `attachments` fields of the message to indicate that it has been deleted. This could potentially be a security issue if the original content of the message was sensitive and should not be retrievable after deletion. Consider removing these fields entirely instead of setting them to a default value.
```diff
-        text: 'This message has been deleted',
-        attachments: []
+        text: null,
+        attachments: null
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->